src/sql/changes/2025-06-02-add-totp-tables.sql

```diff
@@ -0,0 +1,23 @@
+-- Add TOTP authentication tables
+
+-- TOTP secrets with verification state
+CREATE TABLE user_totp (
+    user_id INTEGER PRIMARY KEY REFERENCES users(user_uid) ON DELETE CASCADE,
+    secret TEXT NOT NULL,
+    verified BOOLEAN DEFAULT FALSE,
+    created_at TIMESTAMP WITH TIME ZONE DEFAULT NOW(),
+    updated_at TIMESTAMP WITH TIME ZONE DEFAULT NOW()
```

Comment:
Do we need a trigger to keep track of the `updated_at` column?

```sql
-- Update trigger function to keep track of `updated_at`
CREATE OR REPLACE FUNCTION update_updated_at_column() RETURNS TRIGGER AS $$
BEGIN
    NEW.updated_at = CURRENT_TIMESTAMP;
    RETURN NEW;
END;
$$ LANGUAGE plpgsql;

-- Triggers for TOTP table
CREATE TRIGGER user_totp_updated_at_trigger
BEFORE UPDATE ON user_totp
FOR EACH ROW
EXECUTE FUNCTION update_updated_at_column();
```

Then we don't need to manually update `updated_at`. If we do this we may need to update `clear_functions.sql` to make sure any old triggers are removed properly.

Reply:
Nice! I have added that and updated `clear_functions` to handle that too.